backend/src/core/services/template-client.class.ts

```diff
@@ -115,6 +115,7 @@ export default class TemplateClient {
     } catch (err) {
       logger.error({ message: `${err.stack}` })
       if (err.message.includes('unclosed tag')) throw new TemplateError('There are unclosed curly brackets in the template')
```

Comment:
Maybe: "Check that all your keywords have double curly brackets around them. A correct example is {{ keyword }}, and incorrect ones are { keyword }, {{ keyword } or { keyword }}.  "